src/components/Trends/TrendsPanel.jsx

```diff
@@ -38,16 +39,34 @@ const subLensMap = {
   product: 'Products'
 }
 
+const lensHelperTextMap = {
+  'product': 'Product the consumer identified in the complaint.' +
```

Comment:
👍   i think eslint will nag you for the quote around the keys here though.

Reply:
It first nagged me about the quotes, but then also nagged about consistency for using quotes and not using quotes. I think the latter rule trumped the former but I'll ensure the nag is gone either way.